### DISCUSSION TITLE
Paste Numbers into calculator

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferraridamiano/ConverterNOW#146] [Ideas]
Please allow the user to be able to copy and paste numbers into the calculator.

**Comment by ferraridamiano** [comment]
It is already possible to copy from the calculator, can you confirm this? The issue is just with pasting

**Reply by trymeouteh** [reply]
With more testing done, I was able to copy, just cannot paste

**Reply by ferraridamiano** [reply]
Ok, I will convert this discussion into an issue
